aiohttp/web_response.py

```diff
@@ -440,7 +441,8 @@ class Response(StreamResponse):
 
     def __init__(self, *, body=None, status=200,
                  reason=None, text=None, headers=None, content_type=None,
-                 charset=None):
+                 charset=None, loop=None,
+                 zlib_thread_size=None):
         if body is not None and text is not None:
```

Comment:
I'm not adding a loop param because this was not required before, and if we were to do `self._loop = loop or asyncio.get_event_loop()` it may fail due to there not being an active loop.

Reply:
Fine with it